### PR TITLE
Detail Video

### PR DESCRIPTION
**Pull request by loemhuolay**
Design template for Detail Video
- HTML 
- CSS
- Java Script

For path <`http://localhost:3000/admin/movies/show`>

### CODE REVIEW

File: pj_movie/test/controllers/admin/videos_controller_test.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (N310) [inline pj_movie/test/controllers/admin/videos_controller_test.rb]: In this project, we use Rspec, don't use minitest.
So you should change to rspec test.

File: pj_movie/config/routes.rb [hunk 2]
```diff
@@ -1,6 +1,14 @@
 Rails.application.routes.draw do
 
-  root 'static_pages#home'
-  get 'static_pages/home'
-  get 'static_pages/help'
+  namespace :admin do
+    get 'videos/show'
```
Comment (N310) [inline pj_movie/config/routes.rb]: you should use `resources`.

File: pj_movie/app/controllers/static_pages_controller.rb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 class StaticPagesController < ApplicationController
-  def home
+  def index
```
Comment (N310) [inline pj_movie/app/controllers/static_pages_controller.rb]: Do you need this change in the branch ?
Do you merge `filling-in-layout` ?

File: pj_movie/app/views/static_pages/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<div class="center jumbotron">
```
Comment (N310) [inline pj_movie/app/views/static_pages/index.html.erb]: Do you need to create this change in the branch ?
Do you merge `filling-in-layout` ?

File: pj_movie/app/views/users/edit.html.erb [hunk 5]
```diff
@@ -0,0 +1,21 @@
+<div class="change_pass">
```
Comment (N310) [inline pj_movie/app/views/users/edit.html.erb]: Do you need to create this change in the branch ?

File: pj_movie/app/views/admin/videos/show.html.erb [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+            This my Video Title
+            This my Video Title
+            This my Video Title
+          </div>
+          <div class="views_post">
+            <span>500 views</span>
+            <span>|</span>
+            <span>300 purchases</span>
+            <span>|</span>
+            <span>01/11/2019</span>
```
Comment (N310) [inline pj_movie/app/views/admin/videos/show.html.erb]: you should change this format to `YYYY/MM/DD`.
ex. 2019/11/01

File: pj_movie/app/assets/stylesheets/admin/videos.scss [hunk 7]
```diff
@@ -0,0 +1,78 @@
+.container{
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+.video{
```
Comment (N310) [inline pj_movie/app/assets/stylesheets/admin/videos.scss]: you had better use `SCSS` format.

File: pj_movie/app/assets/stylesheets/admin/videos.scss [hunk 8]
```diff
@@ -0,0 +1,78 @@
+.container{
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
```
Comment (N310) [inline pj_movie/app/assets/stylesheets/admin/videos.scss]: this `.container` is common style in all page ? or it is needed in page of video play ?

File: pj_movie/app/assets/javascripts/admin/script.js [hunk 9]
```diff
@@ -0,0 +1,949 @@
+(function(){function qa(h,p){h.prototype=eb(p.prototype);h.prototype.constructor=h;h.base=p.prototype}function eb(h){function p(){}p.prototype=h;return new p}function Ya(h,p,D){"millisecond"===D?h.setMilliseconds(h.getMilliseconds()+1*p):"second"===D?h.setSeconds(h.getSeconds()+1*p):"minute"===D?h.setMinutes(h.getMinutes()+1*p):"hour"===D?h.setHours(h.getHours()+1*p):"day"===D?h.setDate(h.getDate()+1*p):"week"===D?h.setDate(h.getDate()+7*p):"month"===D?h.setMonth(h.getMonth()+1*p):"year"===D&&h.setFullYear(h.getFullYear()+
```
Comment (N310) [inline pj_movie/app/assets/javascripts/admin/script.js]: do you use `canvasjs` ?

File: pj_movie/app/assets/stylesheets/admin/videos.scss [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+.video_area{
+  height: 500px;
+  justify-content: flex-end;
+  padding: 5%;
+}
+.video_info{
+  display: flex;
+  flex-direction: column;
+}
+.video_title{
```
Comment (N310) [inline pj_movie/app/assets/stylesheets/admin/videos.scss]: you had better check title and views in 681px ~ 830px width.

File: pj_movie/app/assets/javascripts/admin/script.js (hunk 9 above)
Comment (loemhuolay) [reply pj_movie/app/assets/javascripts/admin/script.js]: Yes, I use it.

File: pj_movie/config/routes.rb [hunk 11]
```diff
@@ -1,4 +1,12 @@
 Rails.application.routes.draw do
+  namespace :admin do
+    resources :movies, only: [:show]
+    # resources :movies do
+    #   collection do
+    #     get 'show'
+    #   end
+    # end
```
Comment (PechBunsinh) [inline pj_movie/config/routes.rb]: if you don't need this code you can remove

File: pj_movie/app/assets/javascripts/admin/customize.js [hunk 12]
```diff
@@ -0,0 +1,58 @@
+window.onload = function () {
```
Comment (N310) [inline pj_movie/app/assets/javascripts/admin/customize.js]: Do you need this file?

File: pj_movie/app/assets/javascripts/admin/script.js (hunk 9 above)
Comment (N310) [reply pj_movie/app/assets/javascripts/admin/script.js]: I think you can remove this file.

File: pj_movie/app/assets/stylesheets/admin/movies.scss [hunk 13]
```diff
@@ -0,0 +1,126 @@
+.video_detail_container {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  font-family: Roboto;
```
Comment (N310) [inline pj_movie/app/assets/stylesheets/admin/movies.scss]: Do you need this line?

File: pj_movie/app/views/admin/movies/show.html.erb [hunk 14]
```diff
@@ -0,0 +1,40 @@
...
+        <source src="https://www.radiantmediaplayer.com/media/bbb-360p.mp4" type="video/mp4">
+      </video>
+    </div>
+    <div class="video_info">
+      <div class="video_title_view">
+        <div>
+          <div id="video_title">
+            This best university of the year
+          </div>
+          <button id="edit_title" onclick="get_title()">Edit</button>
```
Comment (N310) [inline pj_movie/app/views/admin/movies/show.html.erb]: I think this button don't need.

File: pj_movie/app/views/admin/movies/show.html.erb [hunk 15]
```diff
@@ -0,0 +1,40 @@
...
+          <span>|</span>
+          <span>300 purchases</span>
+          <span>|</span>
+          <span>2019/11/01</span>
+        </div>
+      </div>
+      <div id="video_descript">
+        It is the only global university performance table to judge research-intensive universities across all of their core missions: teaching, research, knowledge transfer and international outlook. We use 13 carefully calibrated performance indicators to provide the most comprehensive and balanced comparisons, trusted by students, academics, university leaders, industry and governments.
+      </div>
+      <button id="edit_dsc" onclick="get_desc()">Edit</button>
```
Comment (N310) [inline pj_movie/app/views/admin/movies/show.html.erb]: I think this button don't need.

File: pj_movie/app/assets/javascripts/detail_video.js [hunk 16]
```diff
@@ -0,0 +1,22 @@
+function get_title() {
```
Comment (N310) [inline pj_movie/app/assets/javascripts/detail_video.js]: I think you don't need this file.

File: pj_movie/app/controllers/admin/movies_controller.rb [hunk 17]
```diff
@@ -1,4 +1,6 @@
 class Admin::MoviesController < ApplicationController
+  def show
+  end
```
Comment (N310) [inline pj_movie/app/controllers/admin/movies_controller.rb]: You should add empty line.

**Pull request merged**